src/test/util_tests.cpp

```diff
@@ -148,10 +148,14 @@ BOOST_AUTO_TEST_CASE(parse_hex)
     std::vector<unsigned char> result;
     std::vector<unsigned char> expected(std::begin(ParseHex_expected), std::end(ParseHex_expected));
     // Basic test vector
+    result = ToByteVector(ArrayFromHex("04678afdb0fe5548271967f1a67130b7105cd6a828e03909a67962e0ea1f61deb649f6bc3f4cef38c4f35504e51ec112de5c384df7ba0b8d578a4c702b6bf11d5f"));
```

Comment:
nit: this is closely tied to `ParseHex_expected`, if you modify again, consider extracting all 5 usages next to it.

Reply:
To be clear, you are suggesting extracting the repeated string literal "04678afdb0fe5548271967f1a67130b7105c..." into a `constexpr` variable?